#!/usr/bin/python
bounds = [{
    "global_size_x":60.0,
    "global_size_y":22.0,
    "global_size_z":5.5,
    "local_min_x":-30.0,
    "local_min_y":-11.0,
    "local_min_z":-1.0,
    "local_max_x":1.0,
    "local_max_y":11.0,
    "local_max_z":5.0,

},
{
    "global_size_x":70.0,
    "global_size_y":70.0,
    "global_size_z":5.5,
    "local_min_x":-27.5,
    "local_min_y":-3.0,
    "local_min_z":-1.0,
    "local_max_x":1.0,
    "local_max_y":27.5,
    "local_max_z":5.0,

},
{
    "global_size_x":10.0,
    "global_size_y":15.0,
    "global_size_z":5.5,
    "local_min_x":-50.0,
    "local_min_y":-5.5,
    "local_min_z":-1.0,
    "local_max_x":1.0,
    "local_max_y":5.5,
    "local_max_z":5.0,

},
{
    "global_size_x":70.0,
    "global_size_y":22.0,
    "global_size_z":5.5,
    "local_min_x":-45.0,
    "local_min_y":-10.5,
    "local_min_z":-1.0,
    "local_max_x":0.5,
    "local_max_y":10.5,
    "local_max_z":5.0,

},
{
    "global_size_x":60.0,
    "global_size_y":65.0,
    "global_size_z":5.5,
    "local_min_x":-35.0,
    "local_min_y":-32.5,
    "local_min_z":-1.0,
    "local_max_x":1.0,
    "local_max_y":9.0,
    "local_max_z":5.0,

},
{
    "global_size_x":75.0,
    "global_size_y":65.0,
    "global_size_z":5.5,
    "local_min_x":-37.5,
    "local_min_y":-32.0,
    "local_min_z":-1.0,
    "local_max_x":1.0,
    "local_max_y":3.0,
    "local_max_z":5.0,

}]

key_list = list(bounds[0])
